share/smack/top.py

```diff
@@ -135,6 +135,9 @@ def arguments():
   translate_group.add_argument('--entry-points', metavar='PROC', nargs='+',
     default=['main'], help='specify top-level procedures [default: %(default)s]')
 
+  translate_group.add_argument('--memory-safety', action='store_true', default=False,
+    help='enable memory safety checks.')
```

Comment:
Make help message consistent with others. Meaning, remove the period in the end of the message. Other messages do not have it.